### ISSUE TITLE
need to reload the page

### ISSUE THREAD

**Issue by yuyazz** [stcr/subscribe-to-comments-reloaded#457]
hello
i just installed the plugin
i asked someone to write a comment. She received the notification. But when she clicks on the provided link, she see the page without any comment. (comment section disappeared) she needs to relaod the page in order to see it...
is it normal ?? thanks !

**Comment by Reedyseth** [comment]
> . But when she clicks on the provided link, she see the page without any comment. (comment section disappeared) she needs to relaod the page in order to see it...
> is it normal ??

It is not, StCR does not modify the comments section like this. Are you testing this on a dev site ?

**Comment by yuyazz** [comment]
thank you very much for your answer, no it is on my live website
i have hummingbird cache plugin on it..maybe it is the cause of the problem ?

https://www.virtual-trip.fr

she posted the comment on this page
https://www.virtual-trip.fr/trinidad-cuba/



Matthieu Solente

 Intégrateur Developpeur Web Freelance

 T +33 6 36 87 75 23
 yuya1@hotmail.fr<mailto:yuya1@hotmail.fr>
188 Avenue  Jean Lolive- 93 500  Pantin, France
Copier-coller.com<https://www.copier-coller.com/>
[1520593619954_PastedImage]


________________________________
De : Israel Barragan C. <notifications@github.com>
Envoyé : samedi 15 décembre 2018 20:41
À : stcr/subscribe-to-comments-reloaded
Cc : yuyazz; Author
Objet : Re: [stcr/subscribe-to-comments-reloaded] need to reload the page (#457)


. But when she clicks on the provided link, she see the page without any comment. (comment section disappeared) she needs to relaod the page in order to see it...
is it normal ??

It is not, StCR does not modify the comments section like this. Are you testing this on a dev site ?

**Comment by Reedyseth** [comment]
I just left a message on this [post](https://www.virtual-trip.fr/greenwich-village-new-york/)

![image](https://user-images.githubusercontent.com/1228711/50047489-62c7a180-0073-11e9-9893-4c9c6c7c0087.png)

StCR is rendering OK. It might be an issue with your theme or another plugin.

**Comment by yuyazz** [comment]
yes thank you, i answered to the comment..do you still see the comment if you go again on the page?
if it is a bug from my them, i'm sorry for disturbing..

thank you very much for your help

Matthieu Solente

 Intégrateur Developpeur Web Freelance

 T +33 6 36 87 75 23
 yuya1@hotmail.fr<mailto:yuya1@hotmail.fr>
188 Avenue  Jean Lolive- 93 500  Pantin, France
Copier-coller.com<https://www.copier-coller.com/>
[1520593619954_PastedImage]


________________________________
De : Israel Barragan C. <notifications@github.com>
Envoyé : samedi 15 décembre 2018 22:13
À : stcr/subscribe-to-comments-reloaded
Cc : yuyazz; Author
Objet : Re: [stcr/subscribe-to-comments-reloaded] need to reload the page (#457)


I just left a message on this post<https://www.virtual-trip.fr/greenwich-village-new-york/>

[image]<https://user-images.githubusercontent.com/1228711/50047489-62c7a180-0073-11e9-9893-4c9c6c7c0087.png>

StCR is rendering OK. It might be an issue with your theme or another plugin.

**Comment by yuyazz** [comment]
i coded my self the theme, but in the screenshot i can see the comment form div..?
i'll test in on a local site to see

thank you very much!

Matthieu Solente

 Intégrateur Developpeur Web Freelance

 T +33 6 36 87 75 23
 yuya1@hotmail.fr<mailto:yuya1@hotmail.fr>
188 Avenue  Jean Lolive- 93 500  Pantin, France
Copier-coller.com<https://www.copier-coller.com/>
[1520593619954_PastedImage]


________________________________
De : Israel Barragan C. <notifications@github.com>
Envoyé : samedi 15 décembre 2018 22:20
À : stcr/subscribe-to-comments-reloaded
Cc : yuyazz; Author
Objet : Re: [stcr/subscribe-to-comments-reloaded] need to reload the page (#457)


The comment area is not visible, it was remove by another plugin or your theme.

[image]<https://user-images.githubusercontent.com/1228711/50047541-7fb0a480-0074-11e9-9f82-83bc68f7bb0e.png>

**Comment by Reedyseth** [comment]
> but in the screenshot i can see the comment form div..?

Yes, the comment form only holds the input information. You might be having a syntax error or a logical condition that is not fulfill. I will recommend you to debug your theme with a breakpoint in you local environment.

**Comment by yuyazz** [comment]
ok thank you very much for you help, very kind...
i will correct this..thanks to your plugin i realize this bug..

thank you very much for taking time for this.


Matthieu Solente

 Intégrateur Developpeur Web Freelance

 T +33 6 36 87 75 23
 yuya1@hotmail.fr<mailto:yuya1@hotmail.fr>
188 Avenue  Jean Lolive- 93 500  Pantin, France
Copier-coller.com<https://www.copier-coller.com/>
[1520593619954_PastedImage]


________________________________
De : Israel Barragan C. <notifications@github.com>
Envoyé : samedi 15 décembre 2018 22:38
À : stcr/subscribe-to-comments-reloaded
Cc : yuyazz; Author
Objet : Re: [stcr/subscribe-to-comments-reloaded] need to reload the page (#457)


but in the screenshot i can see the comment form div..?

Yes, the comment form only holds the input information. You might be having a syntax error or a logical condition that is not fulfill. I will recommend you to debug your theme with a breakpoint in you local environment.
